242.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {string} s
+ * @param {string} t
+ * @return {boolean}
+ */
+var isAnagram = function (s, t) {
+  const sortedS = s.split("").sort();
+  const sortedT = t.split("").sort();
+
+  if (s.length !== t.length) {
+    return false;
+  }
+
+  for (let i = 0; i < s.length; i++) {
+    if (sortedS[i] !== sortedT[i]) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
You can also do this
```suggestion
  return sortedS.join("") === sortedT.join("");
```